ndcube/ndcube_sequence.py

```diff
@@ -239,6 +245,91 @@ def explode_along_axis(self, axis):
         # creating a new sequence with the result_cubes keeping the meta and common axis as axis
         return self._new_instance(result_cubes, common_axis=new_common_axis, meta=self.meta)
 
+    def _get_sequence_axes_wcs(self):
+        """
+        Returns a WCS object corresponding to the sequence axis of this NDCubeSequence.
+        It can contain multiple pixel axes that map to the same world axis.
+        """
+        table_coords = []
+
+        for axis_name, axis_coords in self.sequence_axis_coords.items():
+            physical_type = self[0].global_coords._internal_coords[axis_name][0]
```

Comment:
Not all global coords are in `_internal_coords`, i.e. ones sliced out of a WCS. I don't think this is the best way to do this.

Wouldn't just `self[0].global_coords.physical_types[axis_name]` work?

Reply:
It does - I think I missed this property while going through the code. Updated it now.